scene/animation/animation_player.cpp

```diff
@@ -1205,9 +1205,9 @@ void AnimationPlayer::play(const StringName &p_name, float p_custom_blend, float
 	_stop_playing_caches();
 
 	c.current.from = &animation_set[name];
-	c.current.pos = p_from_end ? c.current.from->animation->get_length() : 0;
+	c.current.pos = p_from_end ? c.current.from->animation->get_length() : c.current.pos;
```

Comment:
I think it would be more clearer to do:
```
if (p_from_end) {
	c.current.pos = c.current.from->animation->get_length();
}
```
if the `else` branch is meant not to do any change.

Reply:
Agreed, will do it in a sec

Reply:
Changed:

https://github.com/luizcarlos1405/godot/blob/9a2121cf9ce5ba899f5a6537bf5efe84aa91540b/scene/animation/animation_player.cpp#L1207-L1209